### DISCUSSION TITLE
Updating/Contributing to the documentation

### DISCUSSION THREAD

**Discussion by frostybee** [lepoco/wpfui#874] [Q&A] [answered]
Hello,

Is it possible to update/contribute to the documentation? If so, how can I do that? Do I simply add things to the Markdown doc files and submit a pull request? 

Thanks.

**Comment by pomianowski** [comment, marked as answer]
Hey @frostybee, all documentation is in the repository. Single pages are located in the docs directory, https://github.com/lepoco/wpfui/tree/main/docs/documentation.

The rest of the documentation is generated from the code, descriptions and examples of use can be given above the classes.
https://github.com/lepoco/wpfui/blob/7fcc2d8410f056d1d6de97c2f8078b3d8b38539b/src/Wpf.Ui/Markup/FontIconExtension.cs#L11C1-L30C15

The https://wpfui.lepo.co website is automatically updated after merging to the main branch.

Feel free to create a pull request updating the md files or code descriptions, thanks!
